src/Common/src/Interop/Unix/System.Native/Interop.GetUnixVersion.cs

```diff
@@ -0,0 +1,35 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using System.Runtime.InteropServices;
+using System.Text;
+
+internal static partial class Interop
+{
+    internal static partial class Sys
+    {
+        [DllImport(Libraries.SystemNative, CharSet = CharSet.Ansi, SetLastError = true)]
+        private static extern int GetUnixVersion(StringBuilder version, out int capacity);
+
+        internal static string GetUnixVersion()
+        {
+            // max value of _UTSNAME_LENGTH on known Unix platforms is 1024.
+            const int _UTSNAME_LENGTH = 1024;
+            int capacity = _UTSNAME_LENGTH * 3 + 2;
+            StringBuilder version = new StringBuilder(capacity);
```

Comment:
lol, the guidelines says we use var only for cases where it's simple to guess the type, but it doesn't say it's preferred or compulsory. And personally everywhere else in code I have not used var, changing this, will not put me in peace until i change every other instance. So keeping as is.